scripts/install_pecan.sh

```diff
@@ -396,29 +396,24 @@ sudo cp ed_2.1-opt /usr/local/bin/ed2.git
 echo "######################################################################"
 echo "SIPNET"
 echo "######################################################################"
-if [ ! -e ${HOME}/sipnet_unk ]; then
+if [ ! -e ${HOME}/sipnet ]; then
   cd
-  curl -o sipnet_unk.tar.gz http://isda.ncsa.illinois.edu/~kooper/PEcAn/sipnet/sipnet_unk.tar.gz
-  tar zxf sipnet_unk.tar.gz
-  rm sipnet_unk.tar.gz
+  git clone https://github.com/PecanProject/sipnet.git
+  sed -i 's#$(LD) $(LIBLINKS) \(.*\)#$(LD) \1 $(LIBLINKS)#' ${HOME}/sipnet/Makefile
 fi
-cd ${HOME}/sipnet_unk/
+
+cd ${HOME}/sipnet/
+git checkout r136
```

Comment:
do we need a `r136` branch in sipnet repo for this?

Reply:
it is a tag.

Reply:
got it!